sdk/resourcemanager/azure-resourcemanager-redis/CHANGELOG.md

```diff
@@ -3,12 +3,7 @@
 ## 2.10.0-beta.1 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+mgmt, support redisVersion in convenient layer of redis
```

Comment:
Please write similar to this https://github.com/Azure/azure-sdk-for-java/pull/25251

Reply:
And it is feature added section.